### ISSUE TITLE
Documentation issue: setting boundaries on a watershed DEM

### ISSUE THREAD

**Issue by gregtucker** [landlab/landlab#228]
Our Users Guide has no discussion (that I can find) of how to manipulate boundaries on a DEM that represents a watershed. It should have a brief section on how to set NODATA nodes to closed-boundary status, and how to set the watershed outlet node(s) to open boundary status.

The API doc index does have an entry for set_nodata_nodes_to_closed() (actually it has two: one for landlab.grid.ModelGrid, and one for landlab.grid.base.ModelGrid...???)

JSK: I'm assigning this to you so you can bug me or someone else to fix it :)

**Comment by SiccarPoint** [comment]
The wider issue encompasses the fact that we don't have good docs on BCs *at all*. I went through before AGU and (I think) removed actual examples of bad practice, but we don't actually spell out good practice clearly anywhere either. We need:

* A tutorial in a (two?) notebook(s): square grids AND voronois vs one on handling imported grids
* A descriptive section of text on the wiki (including material on all of: regular grids, irregular grids, imported grids)
* The actual functions to be documented clearly in a demarcated section in our planned new code documentation web material.

**Comment by jennyknuth** [comment]
Thanks, Dan, I know you've been aware of this issue and putting in documentation/tutorials where you can. Is it correct to say that BC initialization is now stabilized/standardized and the gap is ready to be filled? If so, I'll make this a priority—update what we have and fill in where needed. 



—

**Comment by SiccarPoint** [comment]
The actual commit is currently being looked at by Eric. Once we get that in, I don’t ind doing the doc update (you might have to remind me).

**Comment by mcflugen** [comment]
Yes, this is PR #210. I'll try to finalize it tonight. It really is just a minor clean-up for the functions that set the status of nodes on the edges of raster grids. I suspect there's additional standardization that still needs to be done.
